Extension/bin/msvc.json

```diff
@@ -28,14 +28,6 @@
       {
         "match": "^/dE--header_only_fallback",
         "replace": "--header_only_fallback"
-      },
-      {
-        "match": "^/lang_c_",
-        "replace": "--c\n--c11"
```

Comment:
if this is being removed in this change, we will need to add the cStandard and cppStandard fields to c_cpp_properties.json.  I was thinking we were going to do this separately though.